src/classes/Recipe.js

```diff
@@ -20,13 +20,15 @@ class Recipe {
   }
 
   returnIngredientsList() {
-    return this.ingredients.map(ingredient => `${ingredient.amount} ${ingredient.unit} ${ingredient.name}`)
+    return this.ingredients.map(ingredient => `${ingredient.amount.toFixed(2).replace(/\.00$/, '')} ${ingredient.unit} ${ingredient.name}`)
```

Comment:
As you do more code reviews, it's good to review your own PR and leave comments about what the code is doing. For instance, Juliana and Paul may not know what this Regex is doing, so you could leave a comment explaining why you had to use it and what it does, which will help them know what's happening 🙂

Reply:
Good point! So, Juliana and Paul, some amounts were displaying like this 0.333333. I added toFixed(2), which puts two digits after the decimal place. But then, I was getting whole numbers with two zeros (e.g., 2.00 eggs), so that is when I had to use replace, which replaces the 00.